src/utils/githubIssue.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const { translateLanguage } = require('../languages/index');
+
+function getGitHubIssueURL(errorMessage) {
+  try {
+    const packageJsonPath = path.join(__dirname, '../../package.json');
```

Comment:
I think you can use `process.cwd()` instead of __dirname here, I think this way you can avoid relative path 

```suggestion
    const packageJsonPath = path.join(process.cwd(), 'package.json');
```

please test it first, I am not sure if it will works